docs/conf.py

```diff
@@ -85,54 +88,51 @@
 # -- General configuration ----------------------------------------------------
 
 # If your documentation needs a minimal Sphinx version, state it here.
-needs_sphinx = "1.7"
-
-# To perform a Sphinx version check that needs to be more specific than
-# major.minor, call `check_sphinx_version("X.Y.Z")` here.
-check_sphinx_version("1.2.1")  # noqa: F405
+needs_sphinx = "3.0"
```

Comment:
This is a big bump, but appears to be fine. What version of sphinx are we actually building with?

Reply:
3.0 is from April 2020 :
https://pypi.org/project/Sphinx/#history